### PR TITLE
Add a tool to merge binpacks

### PR DESCRIPTION
**Pull request by vondele**
this tool with take N binpacks as input to produce 1 binpack as output.
The input binpacks are read in random order, with a probability related to their size,
but each input file is read sequentially. The output is thus an appropriately shuffled binpack.
The tool is much faster than cat'ing the files together followed by a shuffle.
It assumes that the input binpacks themselves have no particular internal ordering.

### CODE REVIEW

File: script/merge_binpacks.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import struct
+import sys
+import os
+import random
+from pathlib import Path
+
+
+def copy_next_chunk(in_file, out_file):
+
```
Comment (Sopel97) [inline script/merge_binpacks.py:9]: spurious new line?

File: script/merge_binpacks.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    # open other args as input file names, and get their sizes
+    in_filenames = []
+    for i in range(1, len(sys.argv) - 1):
+        in_filenames.append(sys.argv[i])
+    print("infiles: ", in_filenames)
+
+    in_files = []
+    in_files_remaining = []
+    for in_filename in in_filenames:
+        in_file = open(in_filename, "rb")
```
Comment (Sopel97) [inline script/merge_binpacks.py:51]: hmm, how does this work with many files? For some compilations the limit might be as low as 256, and I'm not sure if python does anything more to counter that. Either way, probably something to look into in the future if we happen to reach that limit.

**Review by Sopel97 — COMMENTED** [review]
Also, perhaps `[random_]interleave_binpacks.py` would be a better name?

File: script/merge_binpacks.py (hunk 2 above)
Comment (vondele) [reply script/merge_binpacks.py:51]: yes, might be low (1024 by default on linux), but the real use case I have are the very large 16B bitpacks, so 2 - 4 files are typical.

**Pull request merged**